### ISSUE TITLE
Resizing column on second monitor assigns negative values to its width

### ISSUE THREAD

**Issue by PostPollux** [directus/app#1424]
I'm using docker and the latest version here is 7.0.18, which is the Version I am using. So sorry for not using 7.1. If this bug is already fixed, please close this issue.

# Bug Report
There is a problem with resizing the columns in a two monitor setup. It only happens on the second monitor. If my browser is on the first monitor it works just fine. As soon as I move the browser to the second monitor and try to change the width of a column by dragging the small line, the name of the column just disappears and it's very hard to get the text back. When I examined the "directus_collection_presets" in the database I found under "view_options" the following: 
{"tabular":{"widths":{"nickname":200,"punktezahl":-2392,"email":200,"created_by":200,"created_on":200,"modified_by":200}}}

As you can see the width of "punktezahl" is **negative** 2392.

So I guess the function to resize the column should account in some way the current monitor for it's calculation

## Steps to Reproduce

1. You have to use two monitors
2. Create any collection
3. now try to resize the columns
4. now move your browser to the other screen and try to resize the columns again

## Expected Behavior
Resizing should work correctly

## Actual Behavior
Name of the column disappears as it gets a negative value assigned for its width

## Technical Details

- Device: [eg: Desktop,]
- OS: [Windows 10]
- Browser: [Brave Browser 0.61.52]
- Directus App: [eg: 7.0.18]
- Directus API: [eg: 2.0.16]
- Install Method: [docker]

**Comment by rijkvanzanten** [comment]
I think this is a duplicate of https://github.com/directus/app/issues/1240, which was fixed in 7.1 🙂

**Comment by PostPollux** [comment]
Ok, I now tested it by using your demo app instance (https:directus.app) with version 7.1
It's a bit better but still not completely fixed. Maybe it's because the api I connected to is still v 2.0.16.
Now on the second monitor the title of the column still disappears when I start to resize it and it looks as if I can't resize anything, as the only thing that happens is the text disappearing. No resizing actually happening. Now when i release the mouse button to stop resizing at least the name comes back which hadn't been the case with v 7.0.18. But no resizing happened at all in the whole process. On my primary monitor everything works just fine.
